### PR TITLE
usm: javatls: Fix path creation

### PR DESCRIPTION
**Pull request by guyarb**
### What does this PR do?

Fixes java tls path creation on the target namespace.

### Motivation

Fix bugs found during testing.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: pkg/network/protocols/tls/java/hotspot.go [hunk 1]
```diff
@@ -189,7 +189,7 @@ func (h *Hotspot) connect(withCredential bool) (close func(), err error) {
 		return nil, err
 	}
 
-	if err := conn.SetDeadline(time.Now().Add(3 * time.Second)); err != nil {
+	if err := conn.SetDeadline(time.Now().Add(30 * time.Second)); err != nil {
```
Comment (val06) [inline pkg/network/protocols/tls/java/hotspot.go:192]: that sounds a bit large, this code path is called for each java detected java process in a separate go routine right? (`connect` function is an internal method called from `newJavaProcess` which is used as a callback when subscribing for processMonitor on `exec` events)

**Review by val06 — CHANGES_REQUESTED** [review]
cred

File: pkg/network/protocols/tls/java/hotspot.go (hunk 1 above)
Comment (guyarb) [reply pkg/network/protocols/tls/java/hotspot.go]: removed that change for now

**Review by val06 — APPROVED** [review]
non blocking comment: if you make fixes to that function, consider refactoring it a bit to avoid "shadow redeclaration" of `err` variable

**Review by val06 — APPROVED** [review]

**Pull request merged**